README.md

```diff
@@ -58,6 +64,13 @@ Reference contracts that the Helix system uses to make everything trustless.
 Everything comes together here. This is a simulated environment where you can see how to wire
 everything up and even tweak with some parameters for your dApp.
 
-## License
+### [`notebooks`](packages/examples/notebooks/)
+
+Early simulations of the pay-it-backward model.
+
+## Some other notable items...
+[architecture.md](docs/architecture.md) file contains drawings and description of the technical details of the project.

```

Comment:
```suggestion
[Bootleg Architecture](docs/architecture.md) - Drawings and descriptions of the technical details of the project.
```

Though I almost think this should be moved way higher in the doc as it is quite interesting to anyone who's made it to the repo.

Reply:
Yeah, actually... I'll do that now.